Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
@zio-pietro - I changed it back to use the latex quote and an html blockquote (https://github.com/SquareBracketAssociates/UpdatedPharoByExample/commit/828f7603bf030bc2c2576e9490cbb2e3cb74a638 and https://github.com/SquareBracketAssociates/UpdatedPharoByExample/commit/3390f2c3260b781fca1b4aa7bfba9f44e5abcd95). Pillar really needs a dedicated Quote syntax (a multi-line one, either based on indentation like Markdown, or at least with an open/close syntax like Pillar code blocks).

As far as "using Pillar for book writing is a mistake" - I promise you, this is a losing battle, for many reasons :)
Politically, the Powers That Be (the Pharo association, and the actual writers of these books) are very committed to using Pillar instead of, say, Markdown. They are highly unlikely to change their minds on this. (There are several discussions / heated arguments on the Pharo mailing list, on this subject).

But practically, what do you think the book should be written in, instead? Aside from Pillar, there are basically only 3 options:

1) LaTeX - nope. I know you're a skilled LaTeX user, but this is not a realistic format, in terms of crowd-sourced book writing. Aside from a very small handful of CompSci academic types, LaTeX is a nightmare for regular people to use. Also, note that the first edition of Pharo By Example was actually written in LaTeX (there's still a repo for it, linked off the README). And people went through *considerable* effort to rewrite it from LaTeX to Pillar.

2) Markdown (with conversion to HTML and PDF via something like [Pandoc](http://pandoc.org/)).

3) Asciidoc (with conversion to HTML and PDF via [Asciidoctor](http://asciidoctor.org) plugins). 

In terms of options 2 & 3 (markdown and asciidoc).. they're basically equivalent to Pillar. Yes, a bit more people know Markdown (due to Github, etc) than do Pillar. (I would argue that Asciidoc is equally obscure). The capabilities are largely the same, and they're all equally easy to learn. 

Yes, Pillar does have some limitations when compared to Markdown (lack of indentation-based multi-line parsing, for example, for things like Lists, etc). But the surprising thing is, Pillar is moving and evolving way faster than Markdown is. 

So, really, the Pharo community's insistence on using Pillar is deep down about *controlling the means of production*. While Pillar is being actively maintained, we can make feature requests and bug reports (actually, I think @DamienCassou is one of the contributors to Pillar). 

To sum up, the move from Pillar to something else (like Markdown) ain't gonna happen, because a) people have *really* strong opinions in its favor, b) it's essentially equivalent to Markdown anyway, and c) Oh my god, it took lots of man-years of effort to convert the book from LaTeX to Pillar, why would anybody want to convert it to Markdown again? (Although I heard that Pillar renders to Markdown, actually, so never mind that part).